android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -902,6 +916,8 @@ private void handleFullscreen(KrollDict properties)
 
 	public void setSelection(int start, int end)
 	{
+		if (tv == null) return;
```

Comment:
shouldn't you use `getText()` here too?  
Rest looks fine

Reply:
Essentially same effect, since we are using other methods on `tv` next, so it's fine here.